kong/plugins/hmac-auth/daos.lua

```diff
@@ -1,3 +1,9 @@
+local utils = require "kong.tools.utils"
+
+local function random_secret(t)
+  return utils.random_string()
+end
```

Comment:
yeah, silly mistake, we can just set `default = utils.random_string()`. ill fix that